Client/control/screenshot.py

```
@@ -0,0 +1,43 @@
+    """
+    try:
+        with mss.mss() as sct:
+            if region:
+                left, top, width, height = region
+                monitor = {"top": top, "left": left, "width": width, "height": height}
+            else:
+                monitor = sct.monitors[1] # Primary monitor
+
+            sct_img = sct.grab(monitor)
+            return Image.frombytes("RGB", sct_img.size, sct_img.bgra, "raw", "BGRX")
+    except Exception as e:
```

Ruff (0.15.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)